components/prism-json.js

```diff
@@ -8,6 +7,7 @@ Prism.languages.json = {
 		pattern: /"(?:\\.|[^\\"\r\n])*"(?!\s*:)/,
 		greedy: true
 	},
+	'comment': /\/\/.*|\/\*[\s\S]*?(?:\*\/|$)/,
 	'number': /-?\d+\.?\d*(e[+-]?\d+)?/i,
```

Comment:
What happened here?

Reply:
Hmm

Seems I reverted to the incorrect branch and pushed up some changes not part of my commit. Sorry let me fix that.